src/plugins/discover/public/application/embeddable/saved_search_embeddable.tsx

```diff
@@ -168,6 +168,14 @@ export class SavedSearchEmbeddable
     this.searchProps!.isLoading = true;
 
     this.updateOutput({ loading: true, error: undefined });
+    const executionContext = {
+      type: this.type,
+      name: 'discover',
+      id: this.savedSearch.id,
+      description: this.output.title || this.output.defaultTitle || '',
+      url: this.output.editUrl,
+      parent: this.input.executionContext,
+    };
```

Comment:
ok, so because there's a parent, it's clear that it's the saved search embeddable? it might make sense also to append info to the name like `discover - embeddable` since the request here is different? how or where is this info visible?

Reply:
> ok, so because there's a parent, it's clear that it's the saved search embeddable?

Parent indicates that `discover` exists on `dashbord`. Can `discover` exist on `dashboard` in other forms than `saved search embeddable`? I may not be as familiar with this part of the code.

>  it might make sense also to append info to the name like discover - embeddable since the request here is different

`name` is a user-facing name of a Kibana feature. I don't think users are aware of `embeddable` existence. 

>  how or where is this info visible?

Right now, it's available  in two places:
1. in the `debug` logs of `execution_context` service:
```
[DEBUG][execution_context] stored the execution context: {"parent":{"type":"application","name":"dashboard","id":"722b74f0-b882-11e8-a6d9-e546fe2bba5f","description":"[eCommerce] Revenue Dashboard","url":"/view/722b74f0-b882-11e8-a6d9-e546fe2bba5f"},"type":"search","name":"discover","id":"3ba638e0-b894-11e8-a6d9-e546fe2bba5f","description":"[eCommerce] Orders","url":"/app/discover#/view/3ba638e0-b894-11e8-a6d9-e546fe2bba5f"}
```
1. passed as `x-opaque-id` header to Elasticsearch server to be emitted to `search slowlogs` as a string:
```
8fd90dde-9290-4f7b-8181-0acd2b0a0b1d;kibana:application:dashboard:722b74f0-b882-11e8-a6d9-e546fe2bba5f;search:discover:3ba638e0-b894-11e8-a6d9-e546fe2bba5f
```

Reply:
> Parent indicates that `discover` exists on `dashbord`. Can `discover` exist on `dashboard` in other forms than `saved search embeddable`? I may not be as familiar with this part of the code.

no, there's just one, it's not a code issue but a question of naming. I'm good with naming it Discover, since with parent it's clear that it's on a dashboard

thx for explain the other questions, should have checked the description, you were already offering excellent infos about that